### ISSUE TITLE
can't edit claim if tipped (or self supported)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry#1635]
## The Issue
See https://github.com/lbryio/lbry/issues/1635#issuecomment-466033331

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by tzarebczan** [comment]
This was fixed with https://github.com/lbryio/lbry/commit/dd1d1be994fe7b0dd16cda1027494e0e4278b3f2

**Comment by tzarebczan** [comment]
Reopening..this is an issue after you edit the claim a 2nd time...You run into: https://github.com/lbryio/lbry/blob/master/lbrynet/extras/wallet/transaction.py#L66 but the transaction still goes through

```2019-02-21 09:49:33,186 INFO     lbrynet.extras.daemon.Daemon:2066: Success! Published to lbry://Capture5 txid: 1302845ae7507b9fc747bbb110a6feb8e1e42f5ee2d6cb528ebec7b5ce8453a3 nout: 0
2019-02-21 09:49:36,886 INFO     torba.client.baseledger:118: (lbc_mainnet) on_transaction: address=bSi6ewvtPPj7v3CDpwmphxkChveBZrCBbK, height=0, is_verified=False, tx.id=1302845ae7507b9fc747bbb110a6feb8e1e42f5ee2d6cb528ebec7b5ce8453a3
2019-02-21 09:49:36,893 INFO     torba.client.baseledger:118: (lbc_mainnet) on_transaction: address=bMWMyBT2eis9hJavEGeAhsHePaA1qfXQZU, height=0, is_verified=False, tx.id=1302845ae7507b9fc747bbb110a6feb8e1e42f5ee2d6cb528ebec7b5ce8453a3
2019-02-21 09:49:36,899 INFO     torba.client.baseledger:118: (lbc_mainnet) on_transaction: address=bW3h3qre7TcD9nosc8RrUU12Cey5UEF4G9, height=0, is_verified=False, tx.id=1302845ae7507b9fc747bbb110a6feb8e1e42f5ee2d6cb528ebec7b5ce8453a3
2019-02-21 09:49:36,904 INFO     torba.client.baseledger:118: (lbc_mainnet) on_transaction: address=bc4N7iD25BuPFi5cRL8MzoxGw3r3eqn6EB, height=0, is_verified=False, tx.id=1302845ae7507b9fc747bbb110a6feb8e1e42f5ee2d6cb528ebec7b5ce8453a3
2019-02-21 09:52:51,688 WARNING  lbrynet.extras.daemon.Daemon:1929: found 2 claims for the name 5555555
2019-02-21 09:52:51,688 ERROR    lbrynet.extras.daemon.Daemon:493: error handling api request
Traceback (most recent call last):
  File "c:\users\thoma\documents\lbry\lbrynet\extras\daemon\Daemon.py", line 490, in _process_rpc_call
    result = await result
  File "c:\users\thoma\documents\lbry\lbrynet\extras\daemon\Daemon.py", line 2010, in jsonrpc_publish
    claim_dict['stream']['source'] = existing_claims[-1].claim_dict['stream']['source']
  File "c:\users\thoma\documents\lbry\lbrynet\extras\wallet\transaction.py", line 71, in claim_dict
    self._claim_dict = self.claim.claim_dict
  File "c:\users\thoma\documents\lbry\lbrynet\extras\wallet\transaction.py", line 66, in claim
    raise ValueError('Only claim name and claim update have the claim payload.')
ValueError: Only claim name and claim update have the claim payload.```

**Comment by tzarebczan** [comment]
Another user ran into this error recently.

**Comment by eukreign** [comment]
Should be fixed in 0.35.0 (which has a new `publish` and a `stream update` command which avoids looking up claims by name to do the update). Please re-open if it's still a problem.
